### PR TITLE
Add `setAppName`.

### PR DESCRIPTION
**Pull request by alpacaaa**
In a project we're working on we need to be able to set the `appName`, so here's a pull request for it. Let me know if it looks legit.

Thanks for the cool project!

### CODE REVIEW

**Comment by nikolaia** [comment]
Seems like i flipped on travis-ci for the wrong repo, just ignore the failed checks :) /cc @cbrevik

**Comment by cbrevik** [comment]
Hi @alpacaaa and thanks for the PR! 

At first glance this looks fine. I cannot find any docs for this feature though (have you found any?), so will have to test it this evening.

As an aside, I find it kind of strange that for iOS it is set on the tracker, whilst on Android it is set on the analytics instance.

File: android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java [hunk 1]
```diff
@@ -240,4 +240,14 @@ public void setOptOut(Boolean enabled){
...
         }
     }
+
+    @ReactMethod
+    public void setAppName(String trackerId, String appName){
+        GoogleAnalytics analytics = getAnalyticsInstance();
+
+        if (analytics != null)
+        {
+            analytics.setAppName(appName);
```
Comment (cbrevik) [inline android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java]: This method is incorrect, the `setAppName` method is available on the Tracker, not on the Analytics-instance. See another method in the same class, for example `setAnonymizeIp`, on how to retrieve the tracker and call a method on it.

**Comment by cbrevik** [comment]
@alpacaaa I've tested the patch. The implementation is solid except for the Android bit. 
Fix that (see my diff comment) and I'll merge right in. Thanks! 👍 

Rebase if you can :)

**Comment by alpacaaa** [comment]
@cbrevik that was totally stupid :D Should be ok now! Thanks :)

**Comment by cbrevik** [comment]
Very nice @alpacaaa! Thanks for your contribution, I will release a new version after another PR has been merged! 👍

**Comment by alpacaaa** [comment]
Cool! Thanks again for the project! 👍

**Pull request merged**